hodl/exchanges/BittrexHelperAPI.py

```diff
@@ -40,6 +40,15 @@ def print_bittrex_ascii():
           '\____/|_|\__|\__|_|  \___/_/\_\\')
 
 
+def print_bittrex_trade_history():
+    trade_history_data = []
+    trade_history_response = bittrex_v2.get_order_history()
+    if trade_history_response['success'] and trade_history_response['result'] is not None:
+        trade_history_data = trade_history_response['result']
+    print('Trade History for all currencies{}' .format(trade_history_data))
```

Comment:
You only want this to print if the conditions above are true, (therefore, it needs to be indented).

As is currently, when `print_bittrex_trade_history` is called, the print statement will always execute no matter what. We only want to print if the above checks pass.